skipruntime-ts/core/src/skipruntime_helpers.ts

```diff
@@ -144,7 +144,7 @@ function toId(
 ): string {
   const strparams: string[] = [];
   for (const key of Object.keys(params).sort()) {
-    strparams.push(`${key}:${params[key].toString()}`);
+    strparams.push(`${key}:${params[key]!.toString()}`);
```

Comment:
This should have more or less the same behaviour:
```ts
const strparams = Object.entries(params).map(([key, value]) => `${key}:${value.toString()}`).sort();
```

Reply:
Done in new version.